packages/core/package.json

```diff
@@ -32,12 +32,17 @@
     "@ceramicnetwork/common": "^0.15.4",
     "@ceramicnetwork/docid": "^0.4.2",
     "@ceramicnetwork/doctype-caip10-link": "^0.12.6",
+    "@ceramicnetwork/doctype-caip10-link-handler": "^0.12.6",
     "@ceramicnetwork/doctype-tile": "^0.12.8",
+    "@ceramicnetwork/doctype-tile-handler": "^0.12.8",
     "@ceramicnetwork/pinning-aggregation": "^0.1.9",
     "@ceramicnetwork/pinning-ipfs-backend": "^0.1.9",
     "@ethersproject/base64": "^5.0.2",
     "@ethersproject/providers": "^5.0.0-beta.144",
     "@ethersproject/random": "^5.0.2",
+    "@stablelib/random": "^1.0.0",
+    "@stablelib/sha256": "^1.0.0",
+    "ipld-dag-cbor": "^0.17.0",
```

Comment:
Why were these added? I don't see them being used anywhere in your code changes?

Reply:
These are not new dependencies. These are old and used ones, but undeclared.

@stablelib/random: https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/core/src/ceramic.ts#L38
@stablelib/sha256: https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/core/src/dispatcher.ts#L6
ipld-dag-cbor: https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/core/src/dispatcher.ts#L4

Basically, all of the library consumers were at a mercy of npm flaws that allows one to use a dependency required by some unrelated package in this one.

Reply:
ok, thanks. So it was just not included previously which you fixed